frontend/src/pages/projects/screens/detail/connections/ConnectionsTableRow.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { ActionsColumn, Td, Tr } from '@patternfly/react-table';
+import { TableRowTitleDescription } from '~/components/table';
+import { Connection } from './types';
+
+type ConnectionsTableRowProps = {
+  obj: Connection;
+  onEditConnection: (pvc: Connection) => void;
+  onDeleteConnection: (dataConnection: Connection) => void;
+};
+
+const ConnectionsTableRow: React.FC<ConnectionsTableRowProps> = ({
+  obj,
+  onEditConnection,
+  onDeleteConnection,
+}) => (
+  <Tr>
+    <Td dataLabel="Name">
+      <TableRowTitleDescription
+        title={obj.metadata.annotations['openshift.io/display-name']}
+        boldTitle={false}
+        resource={obj}
+        description={obj.metadata.annotations['openshift.io/description']}
+      />
+    </Td>
+    <Td dataLabel="Type">{obj.metadata.annotations['opendatahub.io/connection-type']}</Td>
```

Comment:
Updated. I decided to just pull all the connection types upfront on going to this subsection, since it's going to be used a lot anyways. Then pass it down for wherever it's needed. 
So that network request has been added to the pages `isLoading` and `loadError`, which the page could technically work even if it failed, but it probably makes sense to have still.

![image](https://github.com/user-attachments/assets/3cf659f8-229e-47cc-802d-bb5c225dfb32)